setup.py

```diff
@@ -44,7 +44,7 @@
         'pybtex',
         'pyyaml',
         'requests',
-        'scitools-iris>=2.2',
+        'scitools-iris>=2.2.1',
```

Comment:
```suggestion
        'scitools-iris>=3',
```
and also in package/meta.yaml